### PR TITLE
Fb takyoon handle navigate key screen

### PR DESCRIPTION
**Pull request by takYoon**
handle navigate key screen

### CODE REVIEW

File: src/app/pages/add/create-wallet/create-wallet.ts [hunk 1]
```diff
@@ -104,7 +105,8 @@ export class CreateWalletPage implements OnInit {
     private persistenceProvider: PersistenceProvider,
     private errorsProvider: ErrorsProvider,
     private router: Router,
-    private appProvider: AppProvider
+    private appProvider: AppProvider,
+    private events2: EventsService
```
Comment (vince8x) [inline src/app/pages/add/create-wallet/create-wallet.ts:109]: We should use eventsService

File: src/app/pages/wallet-details/wallet-details.ts [hunk 2]
```diff
@@ -82,7 +83,7 @@ export class WalletDetailsPage {
   public navPramss: any;
   public finishParam: any;
   public isScroll = false;
-
+  isFinish = false;
```
Comment (vince8x) [inline src/app/pages/wallet-details/wallet-details.ts:86]: Should be isFinished? 
Also finish what? Could we use more meaningful name or add comment?

File: src/app/providers/events.service.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+})
+export class EventsService {
+    private refreshKey = new Subject<any>();
+    private refreshWallets = new Subject<any>();
+
+    publishRefresh(data: any) {
+        this.refreshKey.next(data);
+    }
+
+    getRefresh(): Subject<any> {
```
Comment (vince8x) [inline src/app/providers/events.service.ts:15]: Why getRefresh retuen refreshKey? The name seems mismatch?

File: src/app/providers/events.service.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class EventsService {
+    private refreshKey = new Subject<any>();
+    private refreshWallets = new Subject<any>();
```
Comment (vince8x) [inline src/app/providers/events.service.ts:9]: Do we need to unsubscribe the subjects when the services are destroyed?

File: src/app/pages/add/create-wallet/create-wallet.ts (hunk 1 above)
Comment (takYoon) [reply src/app/pages/add/create-wallet/create-wallet.ts]: changed

File: src/app/pages/wallet-details/wallet-details.ts (hunk 2 above)
Comment (takYoon) [reply src/app/pages/wallet-details/wallet-details.ts]: remove unused var

File: src/app/providers/events.service.ts (hunk 3 above)
Comment (takYoon) [reply src/app/providers/events.service.ts]: changed name of function

**Review by vince8x — APPROVED** [review]

**Pull request merged**